lib/ansible/module_utils/redfish_utils.py

```diff
@@ -323,8 +328,11 @@ def get_storage_controller_inventory(self):
         else:
             return {'ret': False, 'msg': "Storage resource not found"}
 
-    def get_disk_inventory(self):
-        result = {'entries': []}
+    def get_multi_storage_controller_inventory(self):
+        return self.aggregate(self.get_storage_controller_inventory)
+
+    def get_disk_inventory(self, systems_uri):
+        result = {}
```

Comment:
A small merge error:
```suggestion
        result = {'entries': []}
```